index.html

```diff
@@ -116,7 +116,32 @@ <h2 class="about-title">About me</h2>
 <p class="about-contend">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 <a href="#">Get My Resume</a>
 <div class="ornament2" ></div>
-
 </section>
+
+
+
+
+<footer class="form">
+<form class="form" action="https://formspree.io/f/xaykaovn" method="post">
+<p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+<input class="form-name" maxlength="30" placeholder="Full name" type="text" name="form_name" id="" required>
+<input class="form-email" placeholder="Email address" type="email" name="form_email" id="" required >
+<textarea class="form-coment" maxlength="500" placeholder="Enter text here" name="form_coment" id="" cols="30" rows="10" required></textarea>
+<button type="submit" >Get in touch</button>
+</form>
+
+
+<ul class="s-media">
+  <li class="form-cat-media"><img src="pictures2/cat.svg" alt=""></li>
+  <li class="in media"><img src="pictures2/linkedin.svg" alt=""></li>
+  <li class="v media"><img src="pictures2/victori.svg" alt=""> </li>
+  <li class="twitter media"><img src="pictures2/twitter2.svg" alt=""></li>
+  <li class="m media"><img src="pictures2/M.svg" alt=""></li>
+</ul>
+<div class="media-underline"></div>
+</footer>
```

Comment:
As requested by the previous reviewer, kindly check and add the line above the social media icons as shown below to match the design

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-09 20-25-27](https://user-images.githubusercontent.com/65233860/200935176-ae85cef5-d1c8-4a5c-8166-fd009d1f9b12.png)  |  ![Screenshot from 2022-11-09 20-30-43](https://user-images.githubusercontent.com/65233860/200935385-6eb38805-b07b-43c0-8485-6604e416296e.png)